render/topologies.go

```diff
@@ -224,9 +223,15 @@ var HostRenderer = MakeReduce(
 // PodRenderer is a Renderer which produces a renderable kubernetes
 // graph by merging the container graph and the pods topology.
 var PodRenderer = MakeReduce(
-	MakeMap(
-		MapContainer2Pod,
-		ContainerRenderer,
+	MakeSilentFilter(
+		func(n report.Node) bool {
+			_, isConnected := n.Latest.Lookup(IsConnected)
+			return n.Topology != Pseudo || isConnected
+		},
+		MakeMap(
+			MapContainer2Pod,
+			ColorConnected(ContainerRenderer),
+		),
```

Comment:
Yeah, good catch.  I guess we don't have an equivalent of "Uncontainer" for containers that aren't part of pods... "Unpodded"? ;-)